daisy_workflows/image_import/suse/translate.py

```diff
@@ -26,67 +26,159 @@
 import utils.diskutils as diskutils
 
 
-network = '''
-BOOTPROTO='dhcp'
-STARTMODE='auto'
-DHCLIENT_SET_HOSTNAME='yes'
-'''
+class Package:
+  name = ''
+  gce = False
+  required = False
+
+  def __init__(self, name, gce, required):
+    self.name = name
+    self.gce = gce
+    self.required = required
+
+
+class SuseRelease:
+  flavor = ''
+  major = ''
+  minor = ''
+  packages = None
+  subscriptions = None
+
+  def __init__(self, flavor, major, minor, packages, subscriptions=None):
+    self.flavor = flavor
+    self.major = major
+    self.minor = minor
+    self.packages = packages
+    self.subscriptions = subscriptions
+
+
+cloud_init = Package('cloud-init', gce=False, required=False)
+gcp_sdk = Package('google-cloud-sdk', gce=True, required=False)
+gce_init = Package('google-compute-engine-init', gce=True, required=True)
+gce_oslogin = Package('google-compute-engine-oslogin', gce=True, required=True)
+
+_distros = [
+    SuseRelease(
+        flavor='opensuse',
+        major='15',
+        minor='1',
+        packages=[cloud_init, gcp_sdk, gce_init, gce_oslogin]
+    ),
+    SuseRelease(
+        flavor='sles',
+        major='15',
+        minor='1',
+        packages=[cloud_init, gcp_sdk, gce_init, gce_oslogin],
+        subscriptions=['sle-module-public-cloud/15.1/x86_64']
+    ),
+    SuseRelease(
+        flavor='sles',
+        major='12',
+        minor='5',
+        packages=[cloud_init, gcp_sdk, gce_init, gce_oslogin],
+        subscriptions=['sle-module-public-cloud/12/x86_64']
+    ),
+]
+
+
+def get_distro(g) -> SuseRelease:
```

Comment:
Add docs to all global methods for easier review and later maintenance.

Reply:
done